helm/oncall/templates/_env.tpl

```diff
@@ -476,6 +488,14 @@
 {{- end }}
 {{- end }}
 
+{{- define "snippet.redis.protocol" -}}
+{{ if not .Values.redis.enabled -}}
+  {{ default "redis" .Values.externalRedis.protocol | quote }}
+{{- else -}}
+  "redis"
+{{- end }}
```

Comment:
can this be simplified to the following?
```suggestion
{{ default "redis" .Values.externalRedis.protocol | quote }}
```

Reply:
Just trying to follow existing structures and code style 😄 Sure I can but please take a look at the chart. It is full of similar solutions: https://github.com/grafana/oncall/blob/dev/helm/oncall/templates/_env.tpl#L271-L301

Let me 15 minutes. Please don't go too far 😄

Reply:
DONE. @joeyorlando please start the tests again, after changes and rebase